intrinsic_funcs/05_vector_integer_arithmetic_functions.md

```diff
@@ -5060,1338 +5060,1338 @@ vint64m4_t vwmulsu_vx_i64m4 (vint32m2_t op1, uint32_t op2, size_t vl);
-vuint64m2_t vqmaccu_vx_u64m2 (vuint64m2_t acc, uint16_t op1, vuint16mf2_t op2, size_t vl);
-vuint64m4_t vqmaccu_vv_u64m4 (vuint64m4_t acc, vuint16m1_t op1, vuint16m1_t op2, size_t vl);
-vuint64m4_t vqmaccu_vx_u64m4 (vuint64m4_t acc, uint16_t op1, vuint16m1_t op2, size_t vl);
-vuint64m8_t vqmaccu_vv_u64m8 (vuint64m8_t acc, vuint16m2_t op1, vuint16m2_t op2, size_t vl);
-vuint64m8_t vqmaccu_vx_u64m8 (vuint64m8_t acc, uint16_t op1, vuint16m2_t op2, size_t vl);
-vint32mf2_t vqmaccsu_vv_i32mf2 (vint32mf2_t acc, vint8mf8_t op1, vuint8mf8_t op2, size_t vl);
-vint32mf2_t vqmaccsu_vx_i32mf2 (vint32mf2_t acc, int8_t op1, vuint8mf8_t op2, size_t vl);
-vint32m1_t vqmaccsu_vv_i32m1 (vint32m1_t acc, vint8mf4_t op1, vuint8mf4_t op2, size_t vl);
-vint32m1_t vqmaccsu_vx_i32m1 (vint32m1_t acc, int8_t op1, vuint8mf4_t op2, size_t vl);
-vint32m2_t vqmaccsu_vv_i32m2 (vint32m2_t acc, vint8mf2_t op1, vuint8mf2_t op2, size_t vl);
-vint32m2_t vqmaccsu_vx_i32m2 (vint32m2_t acc, int8_t op1, vuint8mf2_t op2, size_t vl);
-vint32m4_t vqmaccsu_vv_i32m4 (vint32m4_t acc, vint8m1_t op1, vuint8m1_t op2, size_t vl);
-vint32m4_t vqmaccsu_vx_i32m4 (vint32m4_t acc, int8_t op1, vuint8m1_t op2, size_t vl);
-vint32m8_t vqmaccsu_vv_i32m8 (vint32m8_t acc, vint8m2_t op1, vuint8m2_t op2, size_t vl);
-vint32m8_t vqmaccsu_vx_i32m8 (vint32m8_t acc, int8_t op1, vuint8m2_t op2, size_t vl);
-vint64m1_t vqmaccsu_vv_i64m1 (vint64m1_t acc, vint16mf4_t op1, vuint16mf4_t op2, size_t vl);
-vint64m1_t vqmaccsu_vx_i64m1 (vint64m1_t acc, int16_t op1, vuint16mf4_t op2, size_t vl);
-vint64m2_t vqmaccsu_vv_i64m2 (vint64m2_t acc, vint16mf2_t op1, vuint16mf2_t op2, size_t vl);
-vint64m2_t vqmaccsu_vx_i64m2 (vint64m2_t acc, int16_t op1, vuint16mf2_t op2, size_t vl);
-vint64m4_t vqmaccsu_vv_i64m4 (vint64m4_t acc, vint16m1_t op1, vuint16m1_t op2, size_t vl);
-vint64m4_t vqmaccsu_vx_i64m4 (vint64m4_t acc, int16_t op1, vuint16m1_t op2, size_t vl);
-vint64m8_t vqmaccsu_vv_i64m8 (vint64m8_t acc, vint16m2_t op1, vuint16m2_t op2, size_t vl);
-vint64m8_t vqmaccsu_vx_i64m8 (vint64m8_t acc, int16_t op1, vuint16m2_t op2, size_t vl);
-vint32mf2_t vqmaccus_vx_i32mf2 (vint32mf2_t acc, uint8_t op1, vint8mf8_t op2, size_t vl);
-vint32m1_t vqmaccus_vx_i32m1 (vint32m1_t acc, uint8_t op1, vint8mf4_t op2, size_t vl);
-vint32m2_t vqmaccus_vx_i32m2 (vint32m2_t acc, uint8_t op1, vint8mf2_t op2, size_t vl);
-vint32m4_t vqmaccus_vx_i32m4 (vint32m4_t acc, uint8_t op1, vint8m1_t op2, size_t vl);
-vint32m8_t vqmaccus_vx_i32m8 (vint32m8_t acc, uint8_t op1, vint8m2_t op2, size_t vl);
-vint64m1_t vqmaccus_vx_i64m1 (vint64m1_t acc, uint16_t op1, vint16mf4_t op2, size_t vl);
-vint64m2_t vqmaccus_vx_i64m2 (vint64m2_t acc, uint16_t op1, vint16mf2_t op2, size_t vl);
-vint64m4_t vqmaccus_vx_i64m4 (vint64m4_t acc, uint16_t op1, vint16m1_t op2, size_t vl);
-vint64m8_t vqmaccus_vx_i64m8 (vint64m8_t acc, uint16_t op1, vint16m2_t op2, size_t vl);
+vint32mf2_t vqmacc_vv_i32mf2 (vint32mf2_t acc, vint8mf8_t op1, vint8mf8_t op2, size_t vl, uint8_t tu);
```

Comment:
Not related to this patch, but should quad widening intrinsics be removed since they aren't in the V spec drafts